### ISSUE TITLE
[Request] Suggestion re. pre-filled data

### ISSUE THREAD

**Issue by mariomadproductions** [SabreTools/MPF#463]
When MPF generates its submission info file for a verification, and the user has the program logged into redump, some fields are purposely not pre-filled, or the values have a "(VERIFY THIS) " suffix, to make sure that dumpers check the properties of their dump, and don't inadvertently submit the retrieved data as if it was their own.

However this isn't the case for the following fields that the dumper also needs to check:

- Title
- Category
- Region
- Languages (although this could be marked as "not checked" by the user, since it is per-disc image, not per-disc/release)
- Comments (for the additional serials and other physical properties of the disc, box, etc)

I think this should probably be fixed.

**Comment by mariomadproductions** [comment]
Yeah that makes sense. I think just some kind of clear indication that everything pulled needs to be checked against the actual disc/box. I was thinking something at the top of the txt file itself, but something in the GUI may be better, yeah.

**Comment by mariomadproductions** [comment]
> There are a few fields that automatically get something like "Verify This", generally things that are likely to change. There are other fields, such as language and region, that are prohibitively difficult to do because they rely on static lists of values that it pulls from.

You mean that Verify This can't be prepended to these for some reason? How come? But in that case, it could just be appended to the field name (e.g. `Region (VERIFY THIS):`

**Comment by mnadareski** [comment]
> How about combining this with blocking an ability to submit in the first place if any fields are marked with `VERIFY THIS?`

What is spit out by MPF has to be manually input into Redump. There is no automatic process, therefore, cannot be blocked. There are many users who choose to update the submission info after and don't use the UI to fill it in, so saying that an untouched field is a reason to cancel is also not in the cards.

In the end, the UI can only prevent so much. There is nothing stopping a user from removing Verify This, copying from the site, or anything else. The eventual contract is between the dumper and the mods. This is why I'm opposed to just "add more warnings in the info". It just serves to make it harder for those who do it the right way and another thing to ignore for those who don't.

> You mean that Verify This can't be prepended to these for some reason? How come? But in that case, it could just be appended to the field name (e.g. `Region (VERIFY THIS):`

I'm not doing that. That isn't even something that can be accounted for in the UI for users who do things properly. That forces all users to change the text file after, whether they filled in the information themselves or not. See the above explanation for more details.

**Comment by mnadareski** [comment]
The warning on the login tab was added as of https://github.com/SabreTools/MPF/commit/51f955a14cf742ae784fe3e3f5a5e268ddb1051c

**Comment by mariomadproductions** [comment]
That makes sense. And that warning looks good, thanks.

**Comment by mnadareski** [comment]
Currently, the following items are pulled from Redump, if possible:
- Title, Disc Number/Letter, Disc Title
- Foreign Title
- Category (List of values internally)
- Region (List of values internally)
- Languages (List of values internally)
- Error count
- Version
- Dumper list
- Comments (including splitting into special fields)
- Contents (including splitting into special fields)
- Added date
- Last modified date
